examples/text_classification/ios/TextClassifier/Helper/TextClassifierHelper.swift

```diff
@@ -0,0 +1,32 @@
+// Copyright 2023 The MediaPipe Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import MediaPipeTasksText
+
+class TextClassifierHelper {
+    var textClassifier: TextClassifier?
+
+    init(model: Model) {
+        guard let modelPath = model.modelPath else { return }
+        textClassifier = try? TextClassifier(modelPath: modelPath)
+    }
+
+    func classify(text: String) -> TextClassifierResult? {
+        guard let textClassifier = textClassifier else {
+            print("Text classitifier can not load")
```

Comment:
Is there a way to get the error message here to log for the developer? Rather than just saying it cannot load, we can tell them why.

Reply:
We can show error messages when the creator TextClassifier is in try-catch block. I will change this.